<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>productA</class>
 <widget class="QWidget" name="productA">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>555</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>720</width>
    <height>555</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>720</width>
    <height>555</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="IpLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>IP Adress :</string>
   </property>
  </widget>
  <widget class="QLabel" name="IpText">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>40</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>0.0.0.0</string>
   </property>
  </widget>
  <widget class="QLabel" name="PortLabel">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>40</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Port : </string>
   </property>
  </widget>
  <widget class="QLabel" name="PortText">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>40</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>-1</string>
   </property>
  </widget>
  <widget class="QLabel" name="StateLabel">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>40</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>State : </string>
   </property>
  </widget>
  <widget class="QLabel" name="StateText">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>40</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>NULL</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SendButton">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>450</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Send</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CancelButton">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>450</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>210</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>260</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>310</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_a">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>210</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_b">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>260</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_c">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>310</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="PidLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>PID</string>
   </property>
  </widget>
  <widget class="QLabel" name="PidText">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>20</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="TestLabel">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>20</y>
     <width>50</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>Test</string>
   </property>
  </widget>
  <widget class="QLabel" name="TestLabel2">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>20</y>
     <width>50</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
